test/unit/core.js

```diff
@@ -464,8 +464,11 @@ QUnit.test( "isNumeric", function( assert ) {
 	assert.ok( t( -16 ), "Negative integer number" );
 	assert.ok( t( 0 ), "Zero integer number" );
 	assert.ok( t( 32 ), "Positive integer number" );
+	assert.ok( t( "0b111110" ), "Binary integer literal" );
```

Comment:
Good question. If the test won't run in IE9, we could just test for IE and skip it if so. As for linting, it's okay to use jshint comments for cases like this.